obs_maven/artifact.py

```diff
@@ -228,16 +274,41 @@ def process(self, repo, tmp):
             m = re.match(".*-([^-]+)-[^-]+.[^.]+.rpm", rpm_file)
             if m is None:
                 raise RuntimeError("Failed to get version of " + rpm_file)
-            version = m.group(1)
 
             # Extract the jar and pom
-            (jar, version) = self.extract(rpm_file, tmp)
+            (jar, group, version) = self.extract(rpm_file, tmp, parse_pom)
 
             # Install in the repository
-            self.deploy(jar, version, repo, file.mtime)
+            self.deploy(jar, group, version, repo, file.mtime)
         else:
             logging.info("Skipping artifact %s" % self.artifact)
 
+    @staticmethod
+    def compute_sha1_file(file_name):
+        output_file_name = "{}.sha1".format(file_name)
+        logging.debug("Computing %s" % output_file_name)
+        result = subprocess.run(['sha1sum', file_name], stdout=subprocess.PIPE)
```

Comment:
It would be more pythonic to use [hashlib.sha1()](https://docs.python.org/3/library/hashlib.html#hashlib.sha1). This way you could simply add the `hashlib` to the `setup.py` and the would be no need to document this dependency in the README.